addons/project/static/src/components/subtask_kanban_list/subtask_kanban_create/subtask_kanban_create.js

```diff
@@ -0,0 +1,91 @@
+/** @odoo-module **/
+
+import { Component, useState, useRef, onWillUpdateProps } from "@odoo/owl";
+import { useAutofocus } from "@web/core/utils/hooks";
+
+export class SubtaskCreate extends Component {
+    setup() {
+        this.placeholder = "Add Sub-Task";
```

Comment:
I think it should be 'Sub-tasks', but I cannot find a reliable source backing it up. Anyway, we have a mix of both in the codebase, so maybe it's time to clear it up